libs/service-portal/core/src/components/EventCard/EventCard.treat.ts

```diff
@@ -0,0 +1,46 @@
+import { style } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+
+export const thumbnail = style({
+  backgroundSize: 'contain',
+  backgroundPosition: 'center',
+  backgroundRepeat: 'no-repeat',
+  width: 137,
+  height: 156,
```

Comment:
Would like to see these values come from theme to keep the UI spacing consistent.

Reply:
Sure thing